root/usr/bin/discovery-debug

```diff
@@ -28,6 +28,9 @@ ip addr show
 echo "* ROUTES *"
 ip route
 
+echo "* NEIGHBOURS *"
```

Comment:
interesting, maybe we should considering adding this kind of info via facts too (displaying debug on the UI would be useful too)